src/frontEnd/vueChecks/createSummary.js

```diff
@@ -1,5 +1,212 @@
+    metric: 'SUMMARY-SECURITY-5',
+    description: 'Las contraseñas no son nunca visibles para el usuario despues del login',
+    value: 'Manual'
+  });
+};
+
+const buildingSummary = (summary, reports) => {
+  summary.push({
+    metric: 'SUMMARY-BUILDING-1',
+    description: 'El proyecto utiliza la ultima o anteultima versión del framework',
+    value: !reports.some(
+      elem => elem.metric === generalMetrics.OUTDATED_DEPENDENCIES && elem.value.includes('vue')
+    )
+  });
+
+  summary.push({
+    metric: 'SUMMARY-BUILDING-2',
+    description: 'Las dependencias del proyecto están actualizadas',
+    value:
+      reports.filter(elem => elem.metric === generalMetrics.UNUSED_DEPENDENCIES).length <=
+      limits.maxUnusedDependencies
+  });
+
+  summary.push({
+    metric: 'SUMMARY-BUILDING-3',
+    description: 'Se utiliza el package de deploy para la gestión de releases',
+    value: 'Manual'
+  });
+
+  summary.push({
+    metric: 'SUMMARY-BUILDING-4',
+    description: 'Se corre el checkeo de linter tanto antes de generar un build como de push',
+    value: reports.some(({ metric, value }) => metric === eslintMetrics.ESLINT_ERRORS && !value)
+  });
+
+  summary.push({
+    metric: 'SUMMARY-BUILDING-5',
+    description: 'El proyecto genera archivos minificados en el build',
+    value: 'N/A'
+  });
+
+  summary.push({
+    metric: 'SUMMARY-BUILDING-6',
+    description:
+      'El proyecto usa webpack para generar el build en producción y babel para los imports con alias',
+    value: reports.some(({ metric, value }) => metric === vueMetrics.USE_CLI_SERVICE && value)
+  });
+};
+
+const uiUxSummary = (summary, reports) => {
+  summary.push({
+    metric: 'SUMMARY-UI-UX-1',
+    description: 'El proyecto es mobile friendly según lighthouse',
+    value: reports.some(
+      elem => elem.metric === seoMetrics.LIGHTHOUSE_PWA_OVERALL && elem.value >= limits.pwaMin
+    )
+  });
+
+  summary.push({
+    metric: 'SUMMARY-UI-UX-2',
+    description: 'El proyecto usa Sass respetando el linter correspondiente',
+    value: reports.some(elem => elem.metric === eslintMetrics.ESLINT_CONFIG && elem.value)
+  });
+
+  summary.push({
+    metric: 'SUMMARY-UI-UX-3',
+    description: 'El proyecto posee internacionalización',
+    value: reports.some(elem => elem.metric === vueMetrics.I18N && elem.value >= limits.i18nPercentage)
+  });
+
+  summary.push({
+    metric: 'SUMMARY-UI-UX-4',
+    description: 'Proyecto respeta la estructura de directorios sugerida',
+    value: reports.filter(elem => elem.metric === generalMetrics.FOLDER_STRUCTURE).every(elem => elem.value)
+  });
+
+  summary.push({
+    metric: 'SUMMARY-UI-UX-5',
+    description: 'El proyecto usa vue-loader para generar componentes de un solo archivo SFC',
+    value: reports.some(elem => elem.metric === vueMetrics.USE_VUEX && elem.value)
+  });
+
+  summary.push({
+    metric: 'SUMMARY-UI-UX-7',
+    description: 'SFC que no estén al nivel de app deben estar scoped',
```

Comment:
Rewrite this description. All the SFC should be scoped

Reply:
Cool